client/gutenberg/extensions/tiled-gallery/view.js

```diff
@@ -1,6 +1,134 @@
 /** @format */
 
+/**
+ * External dependencies
+ */
+import { throttle } from 'lodash';
+import ResizeObserver from 'resize-observer-polyfill';
+
 /**
  * Internal dependencies
  */
+import { RESIZE_RATE_IN_MS, DEFAULT_GALLERY_WIDTH } from './constants';
+import { getActiveStyleName, getLayout } from './layouts';
+
+/**
+ * Styles
+ */
 import './view.scss';
+
+const isBrowser = typeof window !== 'undefined' && typeof document !== 'undefined';
+
+// @TODO this isn't working; need a reliable canonical way to determine editor vs view?
+const isEditor = isBrowser && window.hasOwnProperty( 'wp' ) && window.wp.hasOwnProperty( 'editor' );
```

Comment:
This isn't enqueued automatically in the editor by Jetpack anymore, is it? We should be able to drop this now.